<launch>
<!--	<arg name="marker_size" default="4.0" />
	<arg name="max_new_marker_error" default="0.08" />
	<arg name="max_track_error" default="0.2" />

	<arg name="cam_image_topic" default="/camera/depth_registered/points" />
	<arg name="cam_info_topic" default="/camera/depth_registered/camera_info" />		
	<arg name="output_frame" default="/camera_link" />

	 <include file="$(find openni2_launch)/launch/openni2.launch">
  		<param name="depth_registration" type="boolean" value="true" />
   		<arg name="publish_tf" value="true" />
  	</include>
  	
  	-->
  	
	<node pkg="watermellon" type="wm_object_recognition_node" name="wm_object_recognition" output="screen">
			<param name="objects_dir" type="string" value="$(find watermellon)/objects/" />	
			<param name="objects_file" type="string" value="objects.json" />
	</node>

<!--
	<node pkg="rviz" type="rviz" output="screen"
		name="rviz"
		args="-d $(find watermellon)/rviz/wm_object_recognition.rviz"/>-->


</launch>
